presto-druid/src/main/java/com/facebook/presto/druid/metadata/DruidSegmentInfo.java

```diff
@@ -114,38 +123,37 @@ public DeepStorageType getDeepStorageType()
         return DeepStorageType.valueOf(loadSpecification.get(DEEP_STORAGE_TYPE_KEY).toUpperCase());
     }
 
-    public URI getDeepStoragePath() throws IOException
+    public URI getDeepStoragePath()
     {
         // Logic from https://github.com/apache/druid/blob/63c1746fe40fc40d60dbc77f411e58949c102bfa/indexing-hadoop/src/main/java/org/apache/druid/indexer/JobHelper.java#L776
```

Comment:
Any document or github issue we can link here rather than a piece of source code?

Reply:
I'm removing the link as current code looks self-explained after clean up. Let me know if you prefer a github issue link here.

Reply:
Nice work, I think we can just remove the link.  Thanks!